modules/core/ui/views/src/Plugin/Derivative/FarmViewsMenuLink.php

```diff
@@ -15,16 +15,29 @@
 class FarmViewsMenuLink extends ViewsMenuLink {
 
   /**
-   * Set this in child classes.
+   * Specify the entity type ID. This should be set in child classes.
    *
    * @var string
    *
    * @see \Drupal\farm_ui_views\Plugin\Derivative\FarmAssetViewsMenuLink
    * @see \Drupal\farm_ui_views\Plugin\Derivative\FarmLogViewsMenuLink
+   * @see \Drupal\farm_ui_views\Plugin\Derivative\FarmPlanViewsMenuLink
    * @see \Drupal\farm_ui_views\Plugin\Derivative\FarmQuantityViewsMenuLink
    */
   protected string $entityType;
 
+  /**
+   * Specify the View ID. This should be set in child classes.
+   *
+   * @var string
+   *
+   * @see \Drupal\farm_ui_views\Plugin\Derivative\FarmAssetViewsMenuLink
+   * @see \Drupal\farm_ui_views\Plugin\Derivative\FarmLogViewsMenuLink
+   * @see \Drupal\farm_ui_views\Plugin\Derivative\FarmPlanViewsMenuLink
+   * @see \Drupal\farm_ui_views\Plugin\Derivative\FarmQuantityViewsMenuLink
```

Comment:
I don't think these @see are necessary. It should be trivial to find child classes with a modern IDE, especially these because they are all in the same directory. These strings are just another thing to maintain if they ever change in the future.

Reply:
Agreed. We can remove those. I'll add a commit to the PR.